projects/control-service/projects/job-builder-rootless/Dockerfile.python.vdk

```diff
@@ -4,15 +4,21 @@ ARG base_image=python:3.9-slim
 
 FROM $base_image
 
+ARG UID=1000
+ARG GID=1000
+
 # Set the working directory
 WORKDIR /job
 
-# Make sure base image is python based
+# Validate base image is python based
 RUN python -V
+# Create necessary users and set home directory to /job
+RUN groupadd -r -g $GID vdkgroup && useradd -u $UID -g $GID -r vdkuser && chown -R $UID:$GID /job
```

Comment:
I wonder if it is better to add the user and the group to the base image?

Reply:
You mean just this line?

 I was also wondering. Why are you wondering? What are your arguments? I will keep mine so I do not spoil you :)

Reply:
I guess you want to have better control over the user and the group by the ability to overwrite them through the builder. On the other hand, I don't think we will change them at all.